.evergreen/config.yml

```diff
@@ -1236,6 +1444,22 @@ tasks:
     commands:
       - func: "run-atlas-test"
 
+  - name: "aws-auth-test"
+    commands:
+      - func: bootstrap-mongo-orchestration
+        vars:
+          AUTH: "auth"
+          # TODO: SSL??
```

Comment:
I don't think so? Since you're explicitly setting the value for `ORCHESTRATION_FILE` [this](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/run-orchestration.sh#L34-L55) code block will never run. Therefore, the value for SSL will never be used... at least not for `run-orchestration.sh`